app/Http/Requests/Sharing/DeleteSharingRequest.php

```diff
@@ -20,7 +22,7 @@ class DeleteSharingRequest extends BaseApiRequest
 	 */
 	public function authorize(): bool
 	{
-		return AccessControl::can_upload();
+		return Gate::any(UserPolicy::CAN_UPLOAD, User::class);
```

Comment:
Why `Gate::any` instead of `Gate::check`?

Reply:
because I used to apply `admin` and `can_upload`. But then I realized that it was not really necessary. :)
So I need to set it back to `check()`